include/kernels/gauge_polyakov_loop.cuh

```diff
@@ -0,0 +1,221 @@
+#pragma once
+
+#include <gauge_field_order.h>
+#include <quda_matrix.h>
+#include <index_helper.cuh>
+#include <array.h>
+#include <reduction_kernel.h>
+
+namespace quda {
+
+  template <typename Float_, int nColor_, QudaReconstructType recon_>
+  struct GaugeInsertTimesliceArg : public kernel_param<> {
+    using Float = Float_;
```

Comment:
Can you update this and the other kernels in this PR to use `store_t` for the storage type, `real = typename mapper::type` for the compute type, and only expose the name `real` in the kernel?

Reply:
https://github.com/lattice/quda/commit/58845f78894fa8a26a831a6d0a61b8a50b56a269